### PR TITLE
CLN:Replace format() with fstrings in asv/commands/show.py file

### LINKED ISSUE #1193: Bug:Undefined variable x in asv/commands/show.py file
While changing the PY2 format() function to PY3 f strings, for this particular part I need to make it readable by declaring the variables out of the list and calling them later in the f string but this fails with an error `x is not defined`

from this
```

     values = ["({}, {})".format(util.human_value(x[0], benchmark['unit']),
                                            util.human_value(x[1], benchmark['unit']))
``` 
to
```
     first_unit = util.human_value(x[0], benchmark['unit']) 
     second_unit  = util.human_value(x[1], benchmark['unit'])

     values = ["({first_unit}, {second_unit})"

```

Refer to PR #1191 for more context

### PR DESCRIPTION
**Pull request by dorothykiz1**
xref #1077 
closes #1193

### CODE REVIEW

**Comment by dorothykiz1** [comment]
@datapythonista this PR is ready for your review

File: asv/commands/show.py [hunk 1]
```diff
@@ -153,8 +153,7 @@ def _print_results(cls, conf, commit_hash, result_iter, benchmarks, show_details
 
         log.flush()
 
-        color_print("Commit: {}".format(repo.get_decorated_hash(commit_hash,
-                                                                conf.hash_length)),
+        color_print(f"Commit: {repo.get_decorated_hash(commit_hash,conf.hash_length)}",
```
Comment (datapythonista) [inline asv/commands/show.py:156]:
```suggestion
        color_print(f"Commit: {repo.get_decorated_hash(commit_hash, conf.hash_length)}",
```

File: asv/commands/show.py [hunk 2]
```diff
@@ -208,20 +204,20 @@ def get_stat_info(key):
             values = get_stat_info(key)
 
             if key == 'ci_99':
-                values = ["({}, {})".format(util.human_value(x[0], benchmark['unit']),
-                                            util.human_value(x[1], benchmark['unit']))
+                values = [f"({util.human_value(x[0], benchmark['unit'])}, "
+                          f"{util.human_value(x[1], benchmark['unit'])})"
```
Comment (datapythonista) [inline asv/commands/show.py:208]: This is unreadable.

**Review by datapythonista — COMMENTED** [review]
Looks good, added couple of comments.

File: asv/commands/show.py (hunk 2 above)
Comment (dorothykiz1) [reply asv/commands/show.py:208]: This fails with `x is not defined` when i declare the two variables `util.human_value(x[0], benchmark['unit'])` and `util.human_value(x[1], benchmark['unit'])` out of the list

Comment (dorothykiz1) [reply asv/commands/show.py:208]: Let me do this

Comment (dorothykiz1) [reply asv/commands/show.py:208]: I created Issue #1193 for this particular bug

Comment (datapythonista) [reply asv/commands/show.py:208]: I'm checking again, and it doesn't make any sense to get the `x is not defined` error when moving the values to a variable. Can you make the update, so I can see the code and the error in the CI please?

File: asv/commands/show.py [hunk 3]
```diff
@@ -202,10 +202,10 @@ def get_stat_info(key):
 
         for key in ['repeat', 'number', 'ci_99', 'mean', 'std', 'min', 'max']:
             values = get_stat_info(key)
-
             if key == 'ci_99':
-                values = [f"({util.human_value(x[0], benchmark['unit'])}, "
-                          f"{util.human_value(x[1], benchmark['unit'])})"
+                benchmark_unit1 = util.human_value(x[0], benchmark['unit'])
+                benchmark_unit2 = util.human_value(x[1], benchmark['unit'])
+                values = [f"({benchmark_unit1},{benchmark_unit2}"
```
Comment (dorothykiz1) [inline asv/commands/show.py:208]: @datapythonista kindly review this part,also I don't see the checks here

File: asv/commands/show.py [hunk 4]
```diff
@@ -206,22 +202,22 @@ def get_stat_info(key):
 
         for key in ['repeat', 'number', 'ci_99', 'mean', 'std', 'min', 'max']:
             values = get_stat_info(key)
-
             if key == 'ci_99':
-                values = ["({}, {})".format(util.human_value(x[0], benchmark['unit']),
-                                            util.human_value(x[1], benchmark['unit']))
+                benchmark_unit1 = util.human_value(x[0], benchmark['unit'])
+                benchmark_unit2 = util.human_value(x[1], benchmark['unit'])
+                values = [f"({benchmark_unit1},{benchmark_unit2}"
```
Comment (datapythonista) [inline asv/commands/show.py:208]: This doesn't generate the same string, there are a space and a parenthesis missing.

**Comment by datapythonista** [comment]
@dorothykiz1 do you have time to address the comments?

File: asv/commands/show.py (hunk 3 above)
Comment (dorothykiz1) [reply asv/commands/show.py:208]: Noted let me fix this

File: asv/commands/show.py [hunk 5]
```diff
@@ -69,7 +69,7 @@ def run(cls, conf, commit=None, bench=None, machine=None, env_spec=None,
             machines = [machine]
         else:
             raise util.UserError(
-                "Results for machine '{0} not found".format(machine))
+                f"Results for machine '{machine} not found")
```
Comment (datapythonista) [inline asv/commands/show.py:72]:
```suggestion
                f"Results for machine '{machine}' not found")
```

File: asv/commands/show.py [hunk 6]
```diff
@@ -206,22 +202,22 @@ def get_stat_info(key):
 
         for key in ['repeat', 'number', 'ci_99', 'mean', 'std', 'min', 'max']:
             values = get_stat_info(key)
-
             if key == 'ci_99':
-                values = ["({}, {})".format(util.human_value(x[0], benchmark['unit']),
-                                            util.human_value(x[1], benchmark['unit']))
+                benchmark_unit1 = util.human_value(x[0], benchmark['unit'])
+                benchmark_unit2 = util.human_value(x[1], benchmark['unit'])
+                values = [f"({benchmark_unit1}, {benchmark_unit2})"
```
Comment (datapythonista) [inline asv/commands/show.py:208]:
```suggestion
                values = ["({}, {})".format(util.human_value(x[0], benchmark['unit']),
                                            util.human_value(x[1], benchmark['unit']))
```

**Comment by datapythonista** [comment]
Thanks @dorothykiz1

**Pull request merged**